src/index.ts

```diff
@@ -1,795 +1,26 @@
-  StorageError: {
-    _enum: {
-      DidAlreadyPresent: "Null",
-      DidNotPresent: "Null",
-      DidKeyNotPresent: "DidVerificationKeyRelationship",
-      VerificationKeyNotPresent: "Null",
-      CurrentlyActiveKey: "Null",
-      MaxTxCounterValue: "Null",
-      // new
-      MaxPublicKeysPerDidKeyIdentifierExceeded: "Null",
-      MaxTotalKeyAgreementKeysExceeded: "Null",
-      MaxOldAttestationKeysExceeded: "Null",
-    },
-  },
-};
-
-export const types21: RegistryTypes = {
-  ...types20,
-
-  StorageError: {
-    _enum: {
-      DidAlreadyPresent: "Null",
-      DidNotPresent: "Null",
-      DidKeyNotPresent: "DidVerificationKeyRelationship",
-      VerificationKeyNotPresent: "Null",
-      CurrentlyActiveKey: "Null",
-      MaxTxCounterValue: "Null",
-      MaxPublicKeysPerDidKeyIdentifierExceeded: "Null",
-      // renamed
-      MaxTotalKeyAgreementKeysExceeded: "Null",
-      MaxOldAttestationKeysExceeded: "Null",
-    },
-  },
-  // removed
-  DidUpdateDetails: undefined,
-  DidCreationDetails: {
-    did: "DidIdentifierOf",
-    newKeyAgreementKeys: "DidNewKeyAgreementKeys",
-    newAssertionMethodKey: "Option<DidVerificationKey>",
-    newCapabilityDelegationKey: "Option<DidVerificationKey>",
-    newServiceEndpoints: "Option<ServiceEndpoints>",
-  },
-  DidDetails: {
-    authenticationKey: "KeyIdOf",
-    keyAgreementKeys: "DidKeyAgreementKeys",
-    // renamed
-    capabilityDelegationKey: "Option<KeyIdOf>",
-    // renamed
-    assertionMethodKey: "Option<KeyIdOf>",
-    publicKeys: "DidPublicKeyMap",
-    serviceEndpoints: "Option<ServiceEndpoints>",
-    lastTxCounter: "u64",
-  },
-  DelegateSignatureTypeOf: "DidSignature",
-  ContentType: {
-    _enum: ["application/json", "application/ld+json"],
-  },
-
-  // fix: generics mostly don't work here, but OrderedSet is reduced to a Vec anyway
-  OrderedSet: undefined,
-  Collator: {
-    id: "AccountId",
-    stake: "Balance",
-    // fix
-    delegators: "Vec<Stake>",
-    total: "Balance",
-    state: "CollatorStatus",
-  },
-  Delegator: {
-    // fix
-    delegations: "Vec<Stake>",
-    total: "Balance",
-  },
-};
+import type { OverrideBundleDefinition } from "@polkadot/types/types";
+
+import { types8 } from "./types_8";
+import { types9 } from "./types_9";
+import { types10 } from "./types_10";
+import { types12 } from "./types_12";
+import { types17 } from "./types_17";
+import { types18 } from "./types_18";
+import { types19 } from "./types_19";
+import { types20 } from "./types_20";
+import { types21 } from "./types_21";
+import { types23 } from "./types_23";
+
+export { types8 } from "./types_8";
+export { types9 } from "./types_9";
+export { types10 } from "./types_10";
+export { types12 } from "./types_12";
+export { types17 } from "./types_17";
+export { types18 } from "./types_18";
+export { types19 } from "./types_19";
+export { types20 } from "./types_20";
+export { types21 } from "./types_21";
+export { types23 } from "./types_23";
```

Comment:
you already imported them, so you can directly export them again
Like
```
export { types20, types21, types23, ... }
```

Reply:
9d32f87

Reply:
yes, you are there.